requirements.txt

```diff
@@ -1,6 +1,4 @@
 asgiref==3.2.10
 Django==3.1.2
-Pillow==7.2.0
```

Comment:
Don't remove Pillow. It'll be used for images.

Reply:
Added.